enyo.py

```diff
@@ -25,7 +25,6 @@ def connect(self, name, callback):
 
     def send_message(self, name, args=None):
         """Send a message to Enyo"""
-        script = "enyo.Sugar.sendMessage('" + name + "', "

```

Comment:
A few lines later the variable script is referenced again; won't this generate an error?

Reply:
Let me fix this. Thanks !